docs/spark-qualification-tool.md

```diff
@@ -313,95 +338,184 @@ $SPARK_HOME/bin/spark-shell --jars rapids-4-spark-tools_2.12-<version>.jar
 ```
 
 ## Understanding the Qualification tool output
-Its summary report outputs the following information:
-1. Application ID 
-2. Application duration
-3. SQL/DF duration 
-4. Problematic Duration, which indicates potential issues for acceleration.
-   Some of the potential issues include User Defined Function (UDF) or any Dataset APIs.
+
+The Qualification tool generates the output as CSV/log files. Starting from `version-22.06`, the default
+is to generate the report into two different formats:  CSV/log files; and HTML.
+
+### Text and CSV files
+
+The Qualification tool generates a set of log/CSV files in the output folder. The content of each
+file is summarized in the following two sections.
+
+#### Report Summary
+
+The Qualification tool generates a brief summary that includes the projected application's performance
+if the application is run on the GPU. Beside sending the summary to `STDOUT`, the Qualification tool
+generates _text_ as `rapids_4_spark_qualification_output.log`
+
+The summary report outputs the following information:
+
+1. App Name
+2. App ID 
+3. App Duration: Wall-Clock time measured since the application starts till it is completed.
+   If an app is not completed an estimated completion time would be computed.
+4. SQL DF duration: Wall-Clock time duration that includes only SQL-Dataframe queries.
+5. GPU Opportunity: Wall-Clock time that shows how much of the SQL duration can be accelerated on the GPU.
+6. Estimated GPU Duration: Predicted runtime of the app if it was run on GPU.
+7. Estimated GPU Speed-up: Speed-up factor estimated for the app. Calculated as the ratio between
+   `App Duration` and `Estimated GPU Duration`.
+8. Estimated GPU Time Saved: Estimated Wall-Clock time saved if it was run on the GPU.
+9. Recommendation: Recommendation based on `Estimated Speed-up Factor`, where
+   an app can be "_Strongly Recommended_", "_Recommended_", "_Not Recommended_",
+   or "_Not Applicable_". The latter indicates that the app has job or stage failures.
```

Comment:
Would it be helpful to mention the ranges in  "estimated Speed-up factor" . i.e an app is Strongly Recommended if the "estimated speed-up factor" is >=2.5 and so on. 
@tgravescs - Your thoughts on this. Would this be too much details ?

Reply:
The issue with defining the range of each recommendation is that we will have to update the docs every time we decide to tune the thresholds. The github pages (https://nvidia.github.io/spark-rapids/docs/spark-qualification-tool.html) won't be updated until the next release which may cause some confusions between.